### DISCUSSION TITLE
Conflicting values set for option Signed-By regarding source

### DISCUSSION THREAD

**Discussion by gman8321** [linux-surface/linux-surface#1104] [General]
Using the instructions for Debian/Ubuntu on a Surface Book 1. No matter what I do I get the error

E: Conflicting values set for option Signed-By regarding source https://pkg.surfacelinux.com/debian/ release: /var/lib/extrepo/keys/surface-linux.asc != 
E: The list of sources could not be read.

**Comment by StollD** [comment]
Have you tried the manual setup method?

Quoting from the instructions:

> **Please note that the extrepo integration is not maintained by the linux-surface maintainers!**
> If you encounter issues with the repository that extrepo sets up, please remove it and try following the manual guide first. If that fixes it, please report the issue to extrepo and **NOT** to linux-surface. If it doesn't, it is an issue on our side. In that case, do **NOT** report the issue to extrepo, report it to linux-surface.

Also a duplicate of #1097
